api/pkg/transformer/feast/bigtablestore/encoding_test.go

```diff
@@ -1015,7 +1015,7 @@ func TestEntityKeysToBigTable(t *testing.T) {
 	}
 	for _, tC := range testCases {
 		t.Run(tC.desc, func(t *testing.T) {
-			got := entityKeysToBigTable(tC.project, tC.entityKeys)
+			got := entityKeysToBigTable(tC.project, entitiesToEntityNames(tC.entityKeys))
```

Comment:
can you add test where the entities supplied by user doesn't have same order with the one from feature table response?